### ISSUE TITLE
Path to icons

### ISSUE THREAD

**Issue by Taruu** [pooyagolchian/vue-multiple-themes#1]
The path to the icons is relative, not absolute. If routing is used, then the icon indicates the wrong path
The paths should probably look like this?
```css
@font-face {
  font-family: 'icomoon';
  src:  url('/fonts/icomoon.eot?5gakus');
  src:  url('/fonts/icomoon.eot?5gakus#iefix') format('embedded-opentype'),
  url('/fonts/icomoon.ttf?5gakus') format('truetype'),
  url('/fonts/icomoon.woff?5gakus') format('woff'),
  url('/fonts/icomoon.svg?5gakus#icomoon') format('svg');
  font-weight: normal;
  font-style: normal;
  font-display: block;
}
```

**Comment by pooyagolchian** [comment]
@Taruu  You need to font to the public project folder. 
https://github.com/pooyagolchian/vue-multiple-theme-sample/tree/main/public
You can see the sample project of this package. I change this way if you think it would be a better way to show icons like the SVG icon.
When I have developed this package I thought it would be convenient for the developer needs to change the font by himself. 
https://github.com/pooyagolchian/vue-multiple-theme-sample

**Comment by Taruu** [comment]
The fonts are in the public folder. In the case of routing, the following error occurs :/
![изображение](https://user-images.githubusercontent.com/24312612/156889535-67a8846a-3da0-4ba5-80e4-cdaa2b3fd41e.png)

**Comment by pooyagolchian** [comment]
I updated the package and changed it to a relative path. Please update your package to 1.5.2.
If you would have any problem it's my pleasure to fix the issue.
Thanks for your time and consideration.

**Comment by pooyagolchian** [comment]
@Taruu 
Could you please inform me of the new update that I release?
Did you try it?
If it is fixed, I want to close this issue.
Thanks

**Comment by Taruu** [comment]
The path to the icons is still relative. And for correct operation it must be absolute. I gave an example of what worked for me in the first message >_<

**Comment by pooyagolchian** [comment]
This issue has been fixed with the SVG icon in version > 3.0.0
